src/OowpQuery.php

```diff
@@ -40,32 +40,32 @@ function __construct($query = '')
 
     /* Interfaces */
 
-    public function getIterator()
+    public function getIterator(): \Traversable
     {
         return new \ArrayIterator($this->posts);
     }
 
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->posts[$offset]);
     }
 
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
```

Comment:
@rasmuswinter Could it be that mixed return type was introduced in PHP 8?
https://www.php.net/manual/en/language.types.mixed.php

Reply:
good shout. i'll update the requirement